code/modules/clothing/under/striketeam.dm

```diff
@@ -94,8 +96,13 @@ var/list/deathsquad_uniforms = list()
 					mob_indicator = HOLOMAP_OTHER
 			else
 				continue
-		if(mob_indicator != -1)
-			var/image/I = image('icons/12x12.dmi',"ds[mob_indicator]")
+
+		if(mob_indicator != HOLOMAP_ERROR)
+
+			if(!("deathsquad_\ref[src]_\ref[D]" in holomap_cache))
```

Comment:
you should make this string once as a var, and then use that var the 3 times here, to save annoying the string table and all those \ref[] lookups.